templates/etc/nginx/sites-available/default.conf.j2

```diff
@@ -797,4 +797,4 @@ server {
 {% endif %}
 }
 
-# vim:filetype=nginx
+# vim:filetype=jinja
```

Comment:
Thats the downside of it. My problem was that it messed up the Jinja syntax highlighting and so I thought to change that. Removing it entirely is probably better then.